devops/jobs/BackupProdMckinseyMongoToS3.groovy

```diff
@@ -0,0 +1,107 @@
+/*
+ Backup mongo to s3 bucket
+
+ Variables without defaults are marked (required) 
+ 
+ Variables consumed for this job:
+    * SECURE_GIT_CREDENTIALS: secure-bot-user (required)
+    * NOTIFY_ON_FAILURE: alert@example.com
+    * FOLER_NAME: folder
+    * SYS_ADMIN_REPO: repo where the mongo backup script is located (required)
+    * STATUS_BRANCH: default is master
+
+    This job expects the following credentials to be defined on the folder
+    tools-edx-jenkins-aws-credentials: file with key/secret in boto config format
+    mongohq-backups-prod-mckinsey-role-arn: the role to aws sts assume-role
+    mckinsey-mongo-db-password: the password for the mongo databases
+
+*/
+
+package devops.jobs
+import static org.edx.jenkins.dsl.Constants.common_logrotator
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+
+
+class BackupProdMckinseyMongoToS3 {
+
+    public static def job = { dslFactory, extraVars ->
+        dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/backup-prod-mckinsey-mongo-to-s3") {
```

Comment:
Just realized this, sorry I didn't see it earlier.  We should name this something generic like backup-mongo-to-s3 and make it possible to extraVars.get() our internal name for it being mckinsey